tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -62,9 +62,9 @@ sub run {
     my ($self, $run_args) = @_;
     my $provider_setting = get_required_var('PUBLIC_CLOUD_PROVIDER');
 
-    if (is_azure()) {
-        my %maintenance_vars = qesap_az_calculate_address_range(slot => get_required_var('WORKER_ID'));
-        set_var("VNET_ADDRESS_RANGE", $maintenance_vars{vnet_address_range});
+    if (is_azure() || is_ec2()) {
```

Comment:
Why don't we drop this `if` already? Won't we need the same for GCP?

Reply:
here calling it for Gce result in calling `set_var` for settings that eventually gce jobs will not use for the moment. It is ok to do.